src/command_modules/azure-cli-network/azure/cli/command_modules/network/_params.py

```diff
@@ -276,8 +277,9 @@ def completer(prefix, action, parsed_args, **kwargs): # pylint: disable=unused-a
 
 register_cli_argument('network public-ip create', 'name', completer=None)
 register_cli_argument('network public-ip create', 'dns_name', validator=process_public_ip_create_namespace)
-register_cli_argument('network public-ip create', 'public_ip_address_type', help=argparse.SUPPRESS)
+register_cli_argument('network public-ip create', 'dns_name_type', help=argparse.SUPPRESS)
 register_cli_argument('network public-ip create', 'allocation_method', choices=choices_ip_allocation_method, type=str.lower)
+register_cli_argument('network public-ip create', 'version', choices=choices_ip_address_version, type=str.lower)
```

Comment:
no test for ipv6 since it is a simple property set.  The other change was a typo fix, it is not an actual template change.  I uploaded the recording for the IP test.

---
In reply to: [78779812](https://github.com/Azure/azure-cli/pull/877#discussion_r78779812) [](ancestors = 78779812)